crates/esplora/src/blocking_ext.rs

```diff
@@ -50,6 +50,19 @@ pub trait EsploraExt {
     /// The full scan for each keychain stops after a gap of `stop_gap` script pubkeys with no associated
     /// transactions. `parallel_requests` specifies the max number of HTTP requests to make in
     /// parallel.
+    ///
+    /// ## Note
+    ///
+    /// `stop_gap` is defined as "the maximum number of consecutive unused addresses".
+    /// For example, with `stop_gap = 3`, `full_scan` will keep searching for script pubkeys
+    /// until it encounters 3 consecutive script pubkeys with no associated transactions.
+    ///
+    /// This follows the same approach as other Bitcoin-related software,
+    /// such as [Electrum](https://electrum.readthedocs.io/en/latest/faq.html#what-is-the-gap-limit),
+    /// [BTCPay Server](https://docs.btcpayserver.org/FAQ/Wallet/#the-gap-limit-problem),
+    /// and [Sparrow](https://www.sparrowwallet.com/docs/faq.html#ive-restored-my-wallet-but-some-of-my-funds-are-missing).
+    ///
+    /// `stop_gap = 0` will be treated as `stop_gap = 1` for all purposes.
```

Comment:
```suggestion
    /// A `stop_gap` of 0 will be treated as a `stop_gap` of 1.
```

I actually don't see a problem with the way stop_gap was already defined, but I guess it doesn't hurt to add clarifying details. Links to external sources seem like a nice addition.

My only suggestion is to use more plain words in place of pseudo expressions like `stop_gap = 0`.

Reply:
Done in b9ba9b5978c2dfb1ee4cd29ed565365bd62ee75f